components/Article.tsx

```diff
@@ -54,7 +55,7 @@ const Article: React.FunctionComponent<Props> = props => {
         )
     }
     return (
-        <PageLayout contentTitle={props.title} metaTags={metaTags} leftColumn={tocFragment}>
+        <PageLayout contentTitle={props.alternateTitle ? props.alternateTitle : props.title} metaTags={metaTags} leftColumn={tocFragment}>
```

Comment:
Suggestion: would it make sense to simplify this with the `??` operator?

```suggestion
        <PageLayout contentTitle={props.alternateTitle ?? props.title} metaTags={metaTags} leftColumn={tocFragment}>
```

Reply:
Yeah, that makes sense. I will update this.